app/renderer/lib/webview-helpers.js

```diff
@@ -82,6 +82,11 @@ export function parseSource (source) {
   const scripts = $('script');
   scripts.remove();
 
-  // remove all `data-appium-desktop-` prefixes
-  return $.html().replace(/data-appium-desktop-/g, '');
+  return $.html()
```

Comment:
would it be a good idea to also cover this function with a unit test?

Reply:
Yes, that's a good one, will create that later today

Reply:
the $ object is a [cheerio](https://github.com/cheeriojs/cheerio) parser instance. You could check https://stackoverflow.com/questions/21003384/how-to-remove-all-attributes-from-html for example on how to remove attributes from a document

Reply:
Added this